redis.go

```diff
@@ -427,8 +440,12 @@ func (c *baseClient) _process(ctx context.Context, cmd Cmder, attempt int) (bool
 			atomic.StoreUint32(&retryTimeout, 1)
 			return err
 		}
-
-		if err := cn.WithReader(c.context(ctx), c.cmdTimeout(cmd), cmd.readReply); err != nil {
+		readReplyFunc := cmd.readReply
+		// Apply unstable RESP3 search module.
+		if c.opt.Protocol != 2 && c.assertStableCommand(cmd) {
```

Comment:
In this case method should be renamed to `assertUnstableCommand()` to match semantically what we're trying to achieve. Cause if it's unstable we want it to return a raw output, in case if flag was set.